samples/javascript_nodejs/84.bot-authentication-certificate/README.md

```diff
@@ -13,51 +17,174 @@ This bot has been created using [Bot Framework](https://dev.botframework.com/),
+
 ## To try this sample
 
+- Setup ngrok
+  1. Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http --host-header=rewrite 3978
+    ```
+
+- Setup a Bot
+  1. Register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
+
+  2. While registering the bot, use `https://<your_tunnel_domain>/api/messages` as the messaging endpoint.
+      > NOTE: When you create your bot you will create a Microsoft App ID - make sure you keep this for later.
+
 - Clone the repository
 
-```bash
-git clone https://github.com/microsoft/botbuilder-samples.git
-```
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
 
-- In a terminal, navigate to `samples/javascript_nodejs/84.bot-authentication-certificate`
+- Create and configure the SSL/TSL certificate. In this sample we use two possible options to create and set an SSL/TSL certificate. Below is a step-by-step description of each one:
 
-```bash
-cd samples/javascript_nodejs/84.bot-authentication-certificate
-```
+  ### Using local environment
+  - This option requires the following app settings variables:
 
-- Install modules
+    - MicrosoftAppId: App Id of your bot.
 
-```bash
-npm install
-```
+    - CertificateThumbprint: Thumbprint of the certificate uploaded to the app registration.
 
-- Start the bot
+  1. Intall and configure [OpenSSL](https://www.openssl.org/source/) with the latest version
+    - Download the latest version source and add the folder to the [environment variables](https://www.java.com/en/download/help/path.html) path.
+      ```bash
+      setx path "%path%;<OpenSSL path here> 
+      i.e
+      setx path "%path%;C:\Program Files\openssl-3.3.0"
+      ```
 
-```bash
-npm start
-```
+  2. Run the following command in PowerShell
+       - For global environment certificate(Use admin PowerShell) execute:
 
-## Testing the bot using Bot Framework Emulator
+      ```
+      $cert = New-SelfSignedCertificate -CertStoreLocation "." -Subject "CN=<certificate-name>" -KeySpec KeyExchange
+      ```
 
-[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+      ![Global Certificate Command](Images/Local/GlobalCertificateCommand.png)
```

Comment:
Check casing. If repo is cloned in linux/unix system, then `Images` is not the same as `images`.